src/main/java/org/opentripplanner/graph_builder/module/geometry/GeometryAndBlockProcessor.java

```diff
@@ -374,7 +375,7 @@ private boolean hasShapeDist(FeedScopedId shapeId, List<StopTime> stopTimes) {
                     continue;
                 }
                 double distance = segment.distance(coord);
-                if (distance < maxStopToShapeSnapDistance) {
+                if (distance < maxStopToShapeSnapDistance || isFlexTrip) {
```

Comment:
Should we check if either of the stops is a flex stop instead?

Reply:
I'm checking whether the trip contains any flex stops. Do you think that this check is too broad?

Reply:
Yes, we can just check if the variable stop here is an actual stop, and not a flex one